pkgs/applications/misc/crane-gps-watch/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, fetchFromGitHub, libxml2 }:
+
+stdenv.mkDerivation rec {
+  name = "crane-gps-watch";
+  version = "20160308";
+  src = fetchFromGitHub {
+    owner = "mru00";
+    repo = "crane_gps_watch";
+    rev = "221f99dc2925a07658f7b6e01789ced1a18cc08c";
+    sha256 = "0ji50yg85a114pdvw8k5jgfxq312frlxq67bazfaws9mkhnkpi9v";
+  };
+
+meta = with stdenv.lib; {
+  homepage = "https://www.github.com/mru00/crane_gps_watch";
+  description = "Crane GPS Watch client for Linux and Windows";
```

Comment:
I think mentioning platforms in the description is redundant, given `meta.platforms` --- and in this case it is also inaccurate. It doesn't help the user to know that the software works on Windows, if the expression will only evaluate on Linux.